### PR TITLE
Fixed the motor ports for drivebase

### PR DESCRIPTION
**Pull request by NogaRoze**

### CODE REVIEW

File: src/main/java/frc/robot/MotorController.java [hunk 1]
```diff
@@ -35,6 +35,7 @@ public MotorController(String name, int deviceID, int smartCurrentLimit) {
         this(name, deviceID); //calls the constructor above
         //Current limiting is required to prevent brown outs
         mSparkMax.setSmartCurrentLimit(smartCurrentLimit);
+        mSparkMax.setOpenLoopRampRate(.1);
```
Comment (azvanderpas) [inline src/main/java/frc/robot/MotorController.java:38]: Did this have a noticeable effect on driving? 
API doc says "Time in seconds to go from 0 to full throttle."  
100 ms seems faster than the motors are capable of

File: src/main/java/frc/robot/Constants.java [hunk 2]
```diff
@@ -59,8 +59,8 @@
     // Controller in the lab is Logitech F310 USB Gamepad
     public static final int kLeftJoystickAxisX = 0;
     public static final int kLeftJoystickAxisY = 1;
-    public static final int kRightJoystickAxisX = 4;
-    public static final int kRightJoystickAxisY = 5;
+    public static final int kRightJoystickAxisX = 2;
+    public static final int kRightJoystickAxisY = 3;
```
Comment (azvanderpas) [inline src/main/java/frc/robot/Constants.java:63]: Are we certain these are correct?

**Review by azvanderpas — APPROVED** [review]

File: src/main/java/frc/robot/MotorController.java (hunk 1 above)
Comment (AndrewIverson) [reply src/main/java/frc/robot/MotorController.java:38]: It helps make driving more smooth. The ramp rate is for the voltage command for the motor controller, so it takes 100 ms to send full voltage to the motor. The ramp rate is for voltage, or duty cycle, not the velocity of the motor.

File: src/main/java/frc/robot/Constants.java (hunk 2 above)
Comment (AndrewIverson) [reply src/main/java/frc/robot/Constants.java:63]: Yea working in the lab with both different controllers no idea where the previous values came from.

**Pull request merged**